code/game/objects/items/apc_frame.dm

```diff
@@ -24,7 +24,7 @@
 	if (!istype(loc, /turf/simulated/floor))
 		usr << "<span class='warning'>APC cannot be placed on this spot.</span>"
 		return
-	if (A.requires_power == 0 || istype(A, /area/space) || istype(A, /area/mine/unexplored))
+	if (A.requires_power == 0 || istype(A, /area/space) || istype(A, /area/mine/unexplored) || istype(A, /area/mine/explored))

```

Comment:
Yeah, this is not going to affect anything as there is already `istype(A, /area/mine/unexplored)`

Reply:
Nevermind I am plind